### DISCUSSION TITLE
Background tasks: test task has failed

### DISCUSSION THREAD

**Discussion by ovizii** [mealie-recipes/mealie#3442] [Q&A]
Today, I poked around mealie and found the background tasks. I did the test and the result was: test task has failed. The docker logs show nothing helpful.

Any ideas? 

Running mealie v1.4.0

**Comment by ovizii** [comment]
Can anyone else help me clarify what/how to debug?

**Comment by michael-genson** [comment]
Hey there,

Looks like you've stumbled upon an old system from previous versions of Mealie. The background tasks mentioned in that menu aren't actually used anywhere. I'll go ahead and get them removed to avoid confusion. To directly answer your question: it's probably been broken for a long time, it's not really supposed to be accessible

**Reply by ovizii** [reply]
Thanks for explaining.
